src/tqdm_publisher/_progress_handler.py

```diff
@@ -0,0 +1,27 @@
+import queue
+
+from ._progress_subscriber import TQDMProgressSubscriber
+
+
+class TQDMProgressHandler:
```

Comment:
docstrings everywhere would be helpful

Reply:
Sure I'll add these when we dial it in—unless you'd prefer them for initial review.

Reply:
It helps with review to have a least basic descriptors of the intention of your design, yes